modin/_compat/core/py36/pandas_common.py

```diff
@@ -56,18 +58,59 @@ def get_handle(
                 pass
 
 
-def pd_pivot_table(df, **kwargs):  # noqa: PR01, RT01
+def pandas_pivot_table(df, **kwargs):  # noqa: PR01, RT01
     """Perform pandas pivot_table against a dataframe ignoring unsupported args."""
     unsupported_sort = kwargs.pop("sort", None)
     assert unsupported_sort is None
     return df.pivot_table(**kwargs)
 
 
-def pd_convert_dtypes(df, **kwargs):  # noqa: PR01, RT01
+def pandas_convert_dtypes(df, **kwargs):  # noqa: PR01, RT01
     """Perform pandas convert_dtypes against a dataframe or series ignoring unsupported args."""
     unsupported_convert = kwargs.pop("convert_floating", None)
     assert unsupported_convert is None
```

Comment:
```suggestion
    assert (
        unsupported_convert is None
    ), "Unsupported argument passed to DataFrame.convert_dtypes: convert_floating"
```

Reply:
@vnlitvinov perhaps this message was lost among many others :)

Reply:
@vnlitvinov, ping

Reply:
fixed